recipes/snmf/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+apt-get install libswitch-perl -y
```

Comment:
This is not encouraged, please define all dependencies in your meta.yaml file.
We a good documentation here: https://bioconda.github.io/contributing.html that should help you.